app/controller/settings.js

```diff
@@ -0,0 +1,33 @@
+var fs = require('fs');
+var express = require('express');
+
+var router = express.Router();
```

Comment:
then how should I send GET & POST request from this controller (if I can't use router then I can't use router.post & router.get)?